integration-tests/utils/integration-utils.js

```diff
@@ -30,7 +32,7 @@ const findIframe = async page => {
 }
 
 const findInGrid = (page, textContains, options) => {
-  return page.waitForXPath(`//*[@role="table"][contains(normalize-space(.),"${textContains}")]`, options)
+  return page.waitForXPath(`//*[@role="table"][contains(normalize-space(.),"${textContains}")]`, waitForXpathWithOptionVisibleTrue(options))
```

Comment:
As an alternative, this could be
```suggestion
  return page.waitForXPathWithVisibleTrueIfUnspecified(`//*[@role="table"][contains(normalize-space(.),"${textContains}")]`, options)
```

if we refactored the helper to be

```javascript
const waitForXPathWithVisibleTrueIfUnspecified = (page, xpath, options) => page.waitForXPath(xpath, _.defaults({ visible: true }, options)
```

Up to you to decide which one reads better 😊

Reply:
I feel suggested code is confusing, at least true for me. I'd prefer to leave it as-is if you don't feel strongly for change.

Reply:
I don't feel strongly.